schema/cosa/cosa_v1.go

```diff
@@ -147,3 +148,9 @@ type Koji struct {
 type PackageSetDifferences []PackageSetDifferencesItems
 
 type PackageSetDifferencesItems interface{}
+
+type S3 struct {
+	Bucket string `json:"bucket,omitempty"`
+	Path   string `json:"path,omitempty"`
+	URL    string `json:"url,omitempty"`
```

Comment:
Do we need this field? Isn't implied by `bucket` and `path`/`key`?

Reply:
This should be the redirector URL; perhaps the field should be renamed to be more clear?